src/setOfPeople.js

```diff
@@ -39,6 +39,13 @@
  */
 function setOfPeople(people) {
   // write code here
+  const inRoom = new Set();
+  people.forEach(person => (
+    !inRoom.has(person)
```

Comment:
why start with (!inRoom.has(person))? can start start with (inRoom.has(person)) and that'll be easier to read.

Reply:
Beacouse first enter in conditition operator is that there are no one in the room)))).... I simply was writing what I thought)))...